FUNDING.yml

```diff
@@ -0,0 +1,3 @@
+open_collective: standard-readme
+github: RichardLitt
+patron: richlitt
```

Comment:
Shouldn't this read "patreon"?

Reply:
Should do. Want to help be a maintainer?

Reply:
Ideally I would love to, but right now I'm already spread too thin between my day job and various projects. :sweat_smile: I might take you up on that offer sometime in the future, should things calm down in other fronts :)

Reply:
Fair enough! :)